### PR TITLE
Fix folding unfused batchnorm bug

### PR DESCRIPTION
**Pull request by seungjooli**
Folding batch normalization which has parameters like `is_training=False, fuse=False, scale=False` raises an error because its `batch_mean_tensor` is `None`. I suggest using `moving_mean_tensor` instead.

### CODE REVIEW

**Comment by seungjooli** [comment]
I signed it!

**Comment by suharshs** [comment]
Thanks for the fix!

**Review by suharshs — APPROVED** [review]
Looks good. Thanks!

**Comment by tensorflowbutler** [comment]
Nagging Assignee @suharshs: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.
